flatgfa/src/main.rs

```diff
@@ -0,0 +1,32 @@
+use std::env;
+
+use flatgfa::{
+    memfile::{self, *},
+    packedseq::{self, *},
+};
+
+fn main() {
+    let args: Vec<String> = env::args().collect();
+    if args.len() > 2 && args[1] == "import" {
+        let mmap = memfile::map_file(&args[2]); // args[2] is the filename
+        let seq = packedseq::view(&mmap);
+        println!("Sequence: {}", seq);
+    } else if args.len() > 2 && args[1] == "export" {
+        let vec = PackedSeqStore::create(vec![
+            Nucleotide::A,
+            Nucleotide::C,
+            Nucleotide::T,
+            Nucleotide::G,
+        ]);
```

Comment:
Instead of constructing a specific sequence here, let's read a text file from [standard input](https://doc.rust-lang.org/std/io/struct.Stdin.html)! That is, we'd like to do something like this:

```
$ cat myseq.txt | fgfa seq-export myseq.seq
```

where `myseq.txt` contains text like `ACTGTGGACCAATG` or whatever. In other words, let's make a CLI here that can convert between two file types: text files with one nucleotide per character, and our brand-new binary sequence data format.